forest/benchmarking/dfe.py

```diff
@@ -8,16 +8,14 @@
 import numpy as np
 from dataclasses import dataclass
 
-import pyrethrum.operator_estimation as est
+import forest.benchmarking.operator_estimation as est
 from forest.benchmarking.utils import prepare_prod_pauli_eigenstate
 from pyquil import Program
-from pyquil.api import QuantumComputer, get_benchmarker
+from pyquil.api import BenchmarkConnection, QuantumComputer
 from pyquil.operator_estimation import ExperimentSetting, TomographyExperiment, \
     TensorProductState, plusX, minusX, plusY, minusY, plusZ, minusZ
 from pyquil.paulis import PauliTerm, PauliSum, sI, sX, sY, sZ
 
-bm = get_benchmarker()
```

Comment:
What's wrong with initializing here? The benchmarker object doesn't serve any purpose but to perform one of two possible functions that should probably be moved directly into pyquil or benchmarking.

Reply:
In addition to that, IMO it's messy to have a connection object be a module-level variable, and it is much better to explicitly declare its use in the appropriate functions.